build-scripts/Dockerfile.linux-musl-x64

```diff
@@ -5,20 +5,20 @@ ARG GIT_BRANCH='No Branch Info'
 ARG GIT_COMMIT='No Commit Info'
 ARG BUILD_DIR=/build
 ARG TARGET=x86_64-unknown-linux-musl
-# Allow us to override the default `--target-cpu` for the given target triplet
 ARG TARGET_CPU
+ENV RUSTFLAGS="${TARGET_CPU:+${RUSTFLAGS} -Ctarget-cpu=${TARGET_CPU}}"
 WORKDIR /src
 
 COPY . .
 
-RUN apk update && apk add git musl-dev
+RUN echo "RUSTFLAGS: ${RUSTFLAGS}"
```

Comment:
Was this debug print intentionally left here? If so I think it makes sense to add to the other Dockerfiles also

Reply:
ahh, i missed that. i have another small change related to @obycode reply to remove some unneeded comments. 
i'm also going to test updating all debian dockerfiles to debian 12 (currently it's set for 11).

Reply:
unintentional from a copy/paste. it's removed now